packages/manager/cypress/e2e/core/kubernetes/lke-landing-page.spec.ts

```diff
@@ -165,4 +169,186 @@ describe('LKE landing page', () => {
     cy.wait('@getKubeconfig');
     readDownload(mockKubeconfigFilename).should('eq', mockKubeconfigContents);
   });
+
+  it('does not show an Upgrade chip when there is no new kubernetes version', () => {
```

Comment:
```suggestion
  it('does not show an Upgrade chip when there is no new kubernetes standard version', () => {
```
Nit just to try to keep things clear as possible